wp-dependency-installer.php

```diff
@@ -264,7 +267,7 @@ public function admin_init() {
 				} elseif ( $this->is_installed( $slug ) ) {
 					if ( $is_required ) {
 						$this->notices[] = $this->activate( $slug );
-					} else {
+					} elseif ( $is_suggested ) {
 						$this->notices[] = $this->activate_notice( $slug );
```

Comment:
That is, it shows an `activate now` notice only if the plugin is `optional` and the developer only cares that the plugin is `installed` on the system (and not just `always active`)

Reply:
That's how it currently works, with a dismissible notice.